homeassistant/components/esphome/encryption_key_storage.py

```diff
@@ -0,0 +1,93 @@
+"""Encryption key storage for ESPHome devices."""
+
+from __future__ import annotations
+
+import logging
+from typing import TypedDict
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.json import JSONEncoder
+from homeassistant.helpers.singleton import singleton
+from homeassistant.helpers.storage import Store
+from homeassistant.util.hass_dict import HassKey
+
+_LOGGER = logging.getLogger(__name__)
+
+ENCRYPTION_KEY_STORAGE_VERSION = 1
+ENCRYPTION_KEY_STORAGE_KEY = "esphome_encryption_keys"
```

Comment:
```suggestion
ENCRYPTION_KEY_STORAGE_KEY = "esphome.encryption_keys"
```

nit: everything else is `esphome.`